packages/pinia/__tests__/ssr.spec.ts

```diff
@@ -1,6 +1,7 @@
 /**
  * @jest-environment node
```

Comment:
should this be changed too?

Reply:
Good point. Vitest does respect it, but we might as well!